handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/S3Connector.scala

```diff
@@ -18,41 +18,39 @@ import scala.util.Try
 
 object S3Connector extends LazyLogging {
 
-  val bucketName = "emails-from-sf"
-
-  def generateJson(fileIsInS3: Boolean, caseEmail: EmailsFromSfResponse.Records): Either[CustomFailure, String] = {
+  def generateJson(fileIsInS3: Boolean, caseEmail: EmailsFromSfResponse.Records, bucketName: String): Either[CustomFailure, String] = {
```

Comment:
If this function can't fail, it should probably just return a `String` and then lift that into an `Either` at the call site if necessary:
```suggestion
  def generateJson(fileIsInS3: Boolean, caseEmail: EmailsFromSfResponse.Records, bucketName: String): String = {
```

Reply:
This method was actually not being used anywhere, so I've removed it